index.js

```diff
@@ -112,6 +115,7 @@ class KeyringController extends EventEmitter {
         }
         return null
       })
+      .then(this.setUnlocked.bind(this))
```

Comment:
I noticed this is _before_ `persistAllKeyrings` here, but _after_ it for the initial vault creation.

We should probably stick to either before or after, for consistency?